### PR TITLE
Migrate to @babel/parser

### PR DESCRIPTION
**Pull request by skyiea**
Migration https://github.com/wix/okidoc/issues/70

Also contains:
- fix wallaby config
- add test for https://github.com/wix/okidoc/issues/59

### CODE REVIEW

**Comment by bodia-uz** [comment]
Could we split it into few PR?

File: packages/okidoc-md/src/utils/parseSource.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
-import { parse } from 'babylon';
+import { parse } from '@babel/parser';
 import { codeFrameColumns } from '@babel/code-frame';
 
 function parseSource(source) {
   try {
     return parse(source, {
       sourceType: 'module',
       plugins: [
+        'jsx',
```
Comment (bodia-uz) [inline packages/okidoc-md/src/utils/parseSource.js]: lets move this commit to separate PR

**Comment by skyiea** [comment]
> Could we split it into few PR?
> 
> * wallaby
> * babel parser
> * ..

What's the profit?
Those are tiny fixes, split by commits (no need to squash).
